src/lib/analytics/analytics.js

```diff
@@ -43,6 +43,9 @@ export const initAnalytics = async (goodWallet: GoodWallet, userStorage: UserSto
   const identifier = goodWallet && goodWallet.getAccountForType('login')
   const email = userStorage && (await userStorage.getProfileFieldValue('email'))
   log.debug('got identifiers', { identifier, email })
+  if (email && global.mt) {
```

Comment:
there's a lot of global.xxx references in this file.
better to define & initialize all refs to the services services in the constants at the file's beginning and then just check for module constant non-empty 

e.g.
```javascript
import { invoke } from 'lodash'

const Rollbar = global.Rollbar
const Mautic = global.mt
const Amplitude = invoke(global, 'amplitude.getInstance')
// other services

// ...
export const initAnalytics = async (goodWallet: GoodWallet, userStorage: UserStorage) => {
  // ...
  if (Amplitude && Config.amplitudeKey) {
    Amplitude.init(Config.amplitudeKey)
    Amplitude.setVersionName(Config.version)

    // ... 
}

// ...
export const fireMauticEvent = (data: any = {}) => {
  if (!Mautic) return

  const { userId } = Mautic
  let eventData = data

  if (userId) {
    // do not mutate source params
    eventData = { ...data, email: userId }
  }

  Mautic('send', 'pageview', eventData)
}

// adjust other code this way
```

Reply:
as i had the analytics-related tasks #1775 #1866, i've fixed it here: https://github.com/GoodDollar/GoodDAPP/pull/2010/files#diff-d182d40f2ade25618d910d120075488e